src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/requests/makerecalldecisions/UpdateRecommendationAlternativesToRecallRequest.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.integration.requests.makerecalldecisions
+
+fun updateRecommendationAlternativesToRecallRequest() = """
+{
+	"alternativesToRecallTried": {
+		"selected": [{
+				"value": "WARNINGS_LETTER",
+				"details": "We sent a warning letter on 27th July 2022"
+			},
+			{
+				"value": "DRUG_TESTING"
```

Comment:
I think details is mandatory for all values?

Reply:
Added!